### DISCUSSION TITLE
One line condition with braket

### DISCUSSION THREAD

**Discussion by oom-** [biomejs/biome#3108] [Q&A]
Hello, 

I'm having a lot of guard condition in my code similar to : 
```ts
 if (user == null) {
      return apiReplyError(reply, "USER_NOT_FOUND");
}
```

I'm looking to have them formatted on the same line for more clarity: 
```ts
 if (user == null) { return apiReplyError(reply, "USER_NOT_FOUND"); }
```

I would like to keep the curly braces.
I didn't find an option in the formatter to have it this way. 

Does someone knows how to do that ? Thanks?

**Comment by Conaclos** [comment]
You can get this by removing curly brackets:

```js
 if (user == null) return apiReplyError(reply, "USER_NOT_FOUND");
```

**Reply by oom-** [reply]
> I would like to keep the curly braces.

**Comment by ematipico** [comment]
We don't have an option for these cases. If you want to keep the formatting that you wish, I suggest using a suppression comment: https://biomejs.dev/formatter/#ignoring-code
